test/README.md

```diff
@@ -1,22 +1,103 @@
 # Terraform Station Tests
 
-This folder contains all testing done with Station. As of now, these are ran locally from a developer's machine. This folder is supposed to make development and testing much easier. In the future this will run in GHA as well.
+## Overview
 
-## How to run
+This folder contains all tests for the Terraform Station module. Currently, tests are executed locally on a developer's machine, with plans to integrate them into GitHub Actions (GHA) in the future. The aim is to simplify the development and testing process for contributors.
```

Comment:
Is this still correct? @Blinq-SanderBlom

Reply:
I'll update the readme in the other PR where I add the automated tests